use-timescale/schema-management/foreign-data-wrappers.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Foreign data wrappers
+excerpt: Query PostgreSQL databases within or outside Timescale
+products: [cloud, mst, self_hosted]
+keywords: [hypertables, schemas, alter]
+tags: [change]
+---
+
+# Foreign data wrappers
+
+You use foreign data wrappers (FDWs) to query other PostgreSQL databases in a direct, secure, and efficient manner. 
+
+These other databases can be located:
+
+- Within the same Timescale project.
+- Outside of Timescale completely.
+
+Cross-project queries are not supported for security reasons. If using VPC, only services within your VPC can connect.
+
+FDWs are particularly useful if you manage multiple PostgreSQL and time-series instances and need seamless access to both standard and time-series data.
+
+<Procedure>
+
+## Query another Timescale database 
+
+To query another Timescale database in the same project, take the following steps:
+
+1. Enable the FDW extension:
+
+    1. In Timescale Console, select the service and enter the ops mode. 
+    1. Navigate to `Operations` > `Extensions` and find `postgres_fdw` in the list. 
+    1. Expand the section underneath and either click `Run` to install, or copy and run the code in the SQL editor. 
+
+1. Create a foreign data wrapper. 
+
+   A user with the `tsdbadmin` role assigned already has the required `USAGE` permission to create FDWs.
+
+    1. Create a server:
+
+       ```sql
+       CREATE SERVER <server-name> FOREIGN DATA WRAPPER postgres_fdw OPTIONS (host '<service-ID>.<project-id>.tsdb.cloud.timescale.com', dbname '<database-name>', port '<port-number>');
+       ```
+
+    1. Create user mapping:
+
+       ```sql
+       CREATE USER MAPPING FOR <tsdbadmin-user-name> SERVER <server-name> OPTIONS (user '<tsdbadmin-user-name>', password '<tsdbadmin-user-password>');
```

Comment:
<tsdbadmin-user-name> should be just `tsdbadmin`. It's fixed on Cloud and unlikely to change